packages/transaction-controller/src/utils/swaps.ts

```diff
@@ -386,6 +403,7 @@ function updateSwapAndSendTransaction(
     destinationTokenAmount,
     destinationTokenDecimals,
     destinationTokenSymbol,
+    approvalTxParams,
```

Comment:
We're updating this new property just for the existing `swapAndSend` type, but the new `swapAndSendApprovalType` doesn't accept this property, is that intentional?